lib/Jit/CMakeLists.txt

```diff
@@ -38,14 +38,23 @@ if (WIN32)
     ${LLILCJIT_EXPORTS_DEF_TEMP} ${LLILCJIT_EXPORTS_DEF})
 
   set(SHARED_LIB_SOURCES ${SOURCES} ${LLILCJIT_EXPORTS_DEF})
+
+  # For windows we need to specify the CoreCLR environment
+  set(LLILC_TARGET_TRIPLE "${LLVM_DEFAULT_TARGET_TRIPLE}-coreclr")
 else()
   if (UNIX)
     set(LLILCJIT_LINK_LIBRARIES ${LLILCJIT_LINK_LIBRARIES} coreclrpal)
   endif()
 
   set(SHARED_LIB_SOURCES ${SOURCES})
+
+  # For non-windows we can use the default triple for now
+  set(LLILC_TARGET_TRIPLE "${LLVM_DEFAULT_TARGET_TRIPLE}")
 endif()
 
+message(STATUS "LLILC_TARGET_TRIPLE is ${LLILC_TARGET_TRIPLE}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DLLILC_TARGET_TRIPLE=${LLILC_TARGET_TRIPLE}")
```

Comment:
I think you can escape the quotes in order to get the value passed as a string: `"-DLLILC_TARGET_TRIPLE=\"${LLILC_TARGET_TRIPLE}\""`